### PR TITLE
Make tweaks

### PR DESCRIPTION
**Pull request by ismayc**
@mcconvil Let me know if these look OK in the `00-foreword.Rmd` file. I also was a "two spaces after sentences person" :)

### CODE REVIEW

**Comment by mcconvil** [comment]
@ismayc Yes, these look great. 

Sigh, I love two spaces.  It is so hard to let go...

One thing that I should have done:  Can you give me my middle initial (S.) at the end of the foreword and when you mention me at the beginning?

Thanks!

**Comment by mcconvil** [comment]
I love the addition of the hex sticker reference!!

File: 00-foreword.Rmd [hunk 1]
```diff
@@ -1,20 +1,17 @@
...
 
+I first met Dr. Ismay and Dr. Kim while attending their [ModernDive workshop at the 2017 US Conference on Teaching Statistics](https://www.causeweb.org/cause/uscots/uscots17/workshop/3). They pushed us as participants to put data first and to use computers, instead of math, as the engine for statistical inference. That experience helped me modernize my own intro stats course and introduced me to two really forward-thinking statistics and data science educators. It has been exciting to see ModernDive develop and grow into such a wonderful, timely textbook. I hope you have decided to dive on in!
 
 *Kelly McConville, Reed College*
```
Comment (ismayc) [inline 00-foreword.Rmd]:
```suggestion
*Kelly S. McConville, Reed College*
```

**Comment by ismayc** [comment]
Just tweaked them to include the "S." Glad you like the hex reference!

**Comment by mcconvil** [comment]
Perfect!!  Thanks!

**Pull request merged**